### PR TITLE
Add I18n support stafftools/assignment_repos

### PR DESCRIPTION
**Pull request by BelieveC**
Convert static text to symbol for stafftools/assignment_repos views

### CODE REVIEW

File: config/locales/views/stafftools/assignment_repos/en.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+en:
+  views:
+    stafftools:
+      assignment_repos:
+        assignment_title: "%{user_login} assignment repository for %{title}"
```
Comment (nwoodthorpe) [inline config/locales/views/stafftools/assignment_repos/en.yml]: You dropped a 's here after the login.

File: config/locales/views/stafftools/assignment_repos/en.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        created: 'Created'
+        assignment_repo: 'Assignment repository'
+        assignment_repo_small: 'assignment repository'
+        gen_info: 'General information'
```
Comment (nwoodthorpe) [inline config/locales/views/stafftools/assignment_repos/en.yml]: can we change this to `general_info`?

File: config/locales/views/stafftools/assignment_repos/en.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        assignment: 'Assignment'
+        user: 'User'
+        created_on: 'Created on'
+        github_repo: 'GitHub repository'
+        repo_id: 'Repository ID'
+        source_code: 'Source code'
+        dangerzone: 'Dangerzone'
+        delete_repo_title: 'Delete this assignment repository'
+        delete_repo_confirmation: 'Are you ABSOLUTELY sure?'
+        delete_repo_warning: "Unexpected things will happen if you don't read this"
```
Comment (nwoodthorpe) [inline config/locales/views/stafftools/assignment_repos/en.yml]: You dropped a "!" here

**Comment by BelieveC** [comment]
@nwoodthorpe I have made the changes :+1:

**Comment by nwoodthorpe** [comment]
Ran `script/linter` locally

```
==> Running rubocop...
Inspecting 236 files
.....................................................................................................................................
.......................................................................................................

236 files inspected, no offenses detected
==> Running scss-lint...
```

**Pull request merged**